lib/jira/commands/new.rb

```diff
@@ -2,8 +2,15 @@ module Jira
   class CLI < Thor
 
     desc "new", "Creates a new ticket in JIRA and checks out the git branch"
+    method_option :project, aliases: "-p", type: :string, default: nil, banner: "PROJECT"
```

Comment:
these should all be more explicit:
`--project`
`--components`
`--issuetype`
`--parent`
...
single character option names should be reserved for established and known options. otherwise, the option name should be explicit